fairer-iOS/fairer-iOS/Screens/CreateHouseWork/UIComponent/SelectHouseWorkSpaceCollectionView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  SelectHouseWorkSpaceCollectionView.swift
+//  fairer-iOS
+//
+//  Created by LeeSungHo on 2022/09/20.
+//
+
+import UIKit
+
+import SnapKit
+
+final class SelectHouseWorkSpaceCollectionView: BaseUIView {
+    
+    // FIXME
+    private enum Size {
+        static let collectionHorizontalSpacing: CGFloat = 0
+        static let collectionVerticalSpacing: CGFloat = 0
+        static let cellWidth: CGFloat = 105
+        static let cellHeight: CGFloat = 109
+        static let collectionInsets = UIEdgeInsets(
+            top: collectionVerticalSpacing,
+            left: collectionHorizontalSpacing,
+            bottom: collectionVerticalSpacing,
+            right: collectionHorizontalSpacing)
+    }
+    
+    // MARK: - property
+
+    private let collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.sectionInset = Size.collectionInsets
+        flowLayout.itemSize = CGSize(width: Size.cellWidth, height: Size.cellHeight)
+        return flowLayout
+    }()
+    private lazy var collectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout)
+        collectionView.backgroundColor = .clear
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(SelectHouseWorkSpaceCollectionViewCell.self, forCellWithReuseIdentifier: SelectHouseWorkSpaceCollectionViewCell.className)
+        return collectionView
+    }()
+    
+    // MARK: - life cycle
+    
+    override func render() {
+        self.addSubview(collectionView)
+        //FIXME
+        collectionView.snp.makeConstraints {
+            $0.leading.trailing.top.bottom.equalToSuperview()
```

Comment:
p2;
```suggestion
            $0.edges.equalToSuperview()
```
로 바꾸면 더 간단합니다! (메이콴시님께 배운 거 써먹기)

Reply:
넹!!

그런데 요곳은 FIXME가 있듯 아직 레이아웃을 잡지 않은 상태입니다!

메이콴시의 날짜 부분이 오고 나면 레이아웃을 제대로 잡을 수 있을듯 합니다! 후후 이부분은 이 뷰컨의 모든 컴포넌트들이 만들어지면 제대로 수정하겠습니다!